src/flows/components/FlowCard.tsx

```diff
@@ -5,13 +5,15 @@ import {useParams, useHistory} from 'react-router-dom'
 import {ResourceCard} from '@influxdata/clockface'
 import FlowContextMenu from 'src/flows/components/FlowContextMenu'
 import {PROJECT_NAME, PROJECT_NAME_PLURAL} from 'src/flows'
+import {Flow} from 'src/types'
 
 interface Props {
   id: string
-  name: string
+  flow: Flow
 }
 
-const FlowCard: FC<Props> = ({id, name}) => {
+const FlowCard: FC<Props> = ({id, flow}) => {
```

Comment:
i was just poking around in here... all of this prop drilling is.. a bit much. we should be either just accepting the id and using the flow.list context to grab the flow data, or not pass in the id and double down on prop drilling

Reply:
Ya, I was a bit surprised about that the drilling but it seems like part of it is for filtering the sorted data. I can definitely get the flow based on the id though so I'll clean that up